### PR TITLE
Send POI to car

### PR DESCRIPTION
**Pull request by derBroBro**
Hello,

I was not able to find the "send POI to car" in the documentation. Therefore I created this PR. I am hoping all fits to your needs for the readme. 
If something is missing or a change is needed feel free to comment this.

Best Malte

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -654,6 +657,32 @@ An example response for all `POST` commands:
...
+#### Body
+```
+body={
+	"poi": {
+		"city": "abc",
+		"lat": nn.nnn,
+		"lon": nn.nnn,
+		"name": "name",
+		"rating": -1,
+		"postalCode: 12345,
```
Comment (edent) [inline README.md]: I think this should be `"postalCode": "12234",` - not all countries have numeric post codes. Also, note the closing `"` on `"postalCode"`

File: README.md [hunk 2]
```diff
@@ -654,6 +657,32 @@ An example response for all `POST` commands:
...
+Sending a POI to the car.
+
+* `POST /webapi/v1/user/vehicles/:VIN/snedpoi`
+    * Where `:VIN` is your vehicle's VIN.
+    * Remember to include the `Authorization: Bearer` header.
+    * Remember to include the `Content-Type: application/x-www-form-urlencode` header.
+
+#### Body
+```
+body={
```
Comment (edent) [inline README.md:671]: Do you know which of these fields are optional?  Can I put in a City without a lat/long? What happens if I put in a postal code, but use the wrong street name?

File: README.md [hunk 3]
```diff
@@ -654,6 +657,32 @@ An example response for all `POST` commands:
...
+	"poi": {
+		"city": "abc",
+		"lat": nn.nnn,
+		"lon": nn.nnn,
+		"name": "name",
+		"rating": -1,
```
Comment (edent) [inline README.md:677]: Any idea what the rating is, and what values are valid?

**Review by edent — CHANGES_REQUESTED** [review]
Overall very good - thank you! Please can you change the `postalCode` issue? If you know the answer to my other questions, that would be great - but not mandatory.
Thank you so much for this - it is _really_ appreciated :-)

**Comment by derBroBro** [comment]
The postalCode issue should fixed. 
For the lan / lat I am not sure if this is needed. I tried the data from gmaps so this was included by default.
The only point for the rating could be to see the google (or whatever) rating. The location I tested was not rated yet what could result in the "-1".

**Comment by edent** [comment]
Perfect! Dankeschön :-)

**Pull request merged**